bleach/utils.py

```diff
@@ -40,3 +41,72 @@ def force_unicode(text):
 
     # If not, convert it
     return six.text_type(text, "utf-8", "strict")
+
+
+# a subset of Django's URL validator
```

Comment:
If this code is being copied from Django, it should be in a standalone module with the copyright, license information, and some information about which version was copied at the top. Maybe it's better maintenance-wise to put it in `vendor/`.